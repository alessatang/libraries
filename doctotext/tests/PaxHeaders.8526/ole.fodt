30 mtime=1384978086.825497466
29 atime=1389151482.05185051
29 ctime=1389151482.05185051
