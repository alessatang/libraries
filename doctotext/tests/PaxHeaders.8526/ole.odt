30 atime=1389151483.396844841
30 ctime=1389151483.396844841
