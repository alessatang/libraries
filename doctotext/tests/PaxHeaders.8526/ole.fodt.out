30 mtime=1384978086.705497426
30 atime=1389151482.052850506
30 ctime=1389151482.052850506
